easybuild/easyconfigs/q/Qt5/Qt5-5.10.1-foss-2018b.eb

```diff
@@ -24,6 +24,7 @@ dependencies = [
     ('libGLU', '9.0.0'),
     ('NSS', '3.39'),
     ('DBus', '1.13.6'),
+    ('gperf', '3.1'),
```

Comment:
@jhein32 We should apply this same fix in the other easyconfigs for `Qt5` 5.10.1?

Reply:
I only had the issue in foss/2018b.  I could build intel/2018a and foss/2018a without issues. Most likely I have an older version that doesn't force a webengine.  gperf is only needed when the webengine is build - as noted in Slack, I had that failing and found a clear comment in the log: No gperf, no webengine.

Reply:
Ok, since you asked so nicely ;)  I put a revised Qt 5.10.1 for foss/2018a.

Reply:
I leave the intel/2018a config to be fixed once the webengine issues with intel have been fixed.

Reply:
@jhein32 I added `gperf` as a dep in #7010